client/src/tutorials/TinyAdventureTwo/files/client.ts.raw

```diff
@@ -53,7 +53,7 @@ console.log("o........💎");
 // Here we move to the right three times and collect the chest at the end of the level
 for (let i = 0; i < 3; i++) {
   txHash = await pg.program.methods
-    .moveRight()
+    .moveRight("gib")
```

Comment:
This instruction doesn't seem to be taking any parameters in the default code:

https://github.com/solana-playground/solana-playground/blob/be4bcbe25e4364c388377b32300c3f12b8246e76/client/src/tutorials/TinyAdventureTwo/files/lib.rs#L41

The password is mentioned later on in the tutorial in page 3:

https://github.com/solana-playground/solana-playground/blob/be4bcbe25e4364c388377b32300c3f12b8246e76/client/src/tutorials/TinyAdventureTwo/pages/3.md#L20-L23

And the same page also mentions that you need to make this change from client side:

https://github.com/solana-playground/solana-playground/blob/be4bcbe25e4364c388377b32300c3f12b8246e76/client/src/tutorials/TinyAdventureTwo/pages/3.md#L79

Thefore, the default code looks to be correct and this change would break it.